### ISSUE TITLE
Prevent people to abandone then start new games.

### ISSUE THREAD

**Issue by gbtami** [gbtami/pychess-variants#422]
Most of the times [is_playing()](https://github.com/gbtami/pychess-variants/blob/master/server/wsl.py#L20) works OK.
But sometimes not. Why?
![https://i.imgur.com/kz6aPgE.png](https://i.imgur.com/kz6aPgE.png)

**Comment by abunickabhi** [comment]
Yep I agree that there should be some form of a penalty for abandoning games. Like a penalty on the rating, or timing out the player from seeking challenges. It will help weed out random clickers users on the site, who are more random than the random mover bot.

**Comment by Nordlandia** [comment]
This is a well-known dilemma that happens too often. Some kind of restriction should be in place.

**Comment by gbtami** [comment]
![https://i.imgur.com/dAD2SxT.png](https://i.imgur.com/dAD2SxT.png)
